mplc/doc/documentation.md

```diff
@@ -248,14 +248,24 @@ There are 2 ways to select a dataset. You can either choose a pre-implemented da
 
 ![Example of the advanced split option](../../img/advanced_split_example.png)
 
-- `corrupted_datasets`: `[not_corrupted (default), shuffled or corrupted]`  
-  Enables to artificially corrupt the data of one or several partners:
-
-    - `not_corrupted`: data are not corrupted
-    - `shuffled`: each label is randomly shuffled
-    - `corrupted`: each label is offset of one class
-
-  Example: `corrupted_datasets=[not_corrupted, not_corrupted, not_corrupted, shuffled]`
+- `corruption_parameters`: `[{'corruption_method':'permutation'},{}]`  Enables to artificially corrupt the data of one or several partners. The parameter should be a list of dictionary. The possible keys, values are the following:
+    - `'corruption_method'`: Indicating the corruption method to use to corrupt the partner's data. The possibilities are:
+        - `'not_corrupted'` (default): the data remains unchanged. 
```

Comment:
What is the diference between empty dict and {'corruption_method':'not_corrupted'} ?

Reply:
There is not, if the dict is empty the default value 'not corrupted' is selected